generators/Exercises/Generators/PythagoreanTriplet.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Exercism.CSharp.Exercises;
+using Exercism.CSharp.Output;
+using Exercism.CSharp.Output.Rendering;
+using Newtonsoft.Json.Linq;
+
+namespace generators.Exercises.Generators
+{
+    public class PythagoreanTriplet : GeneratorExercise
```

Comment:
You can simplify this generator quite a bit, using the existing generator framework methods:

```csharp
public class PythagoreanTriplet : GeneratorExercise
{
    protected override void UpdateTestMethod(TestMethod testMethod)
    {
        testMethod.Expected = ConvertExpected(testMethod.Expected);
    }

    private dynamic ConvertExpected(dynamic value)
    {
        int[][] values = value.ToObject<int[][]>();
        var triplets = values.Select(ints => (ints[0], ints[1], ints[2])).ToArray();
        return new UnescapedValue(Render.ArrayMultiLine(triplets));
    }

    protected override void UpdateNamespaces(ISet<string> namespaces)
    {
        namespaces.Add(typeof(Array).Namespace);
    }
}
```

This generator should behave the same way as your version and is a bit simpler.

Reply:
Nice! It seems much simpler.
I missed the point that Actual value can be left untouched.
Jagged array also fits very nice here.